lab/lab1/part3-data-transformation.html

```diff
@@ -65,19 +65,15 @@
 
     var data = bikeArrayClean;
 
-    /* =====================
-
-    Start code to filter data
-
-    ===================== */
-
+    var dataFiltered = [];
 
+    for(i = 0; i < bikeArrayClean.length; i++) {
```

Comment:
`Without modifying part4-data-dirty.js, change your code to account for the
    dirty data. Note: it is okay if your new code ignores a few of the points,
    if their formatting is especially wrong. But it should accommodate most points.`

This is the challenging part of this lab and it's important to at least try using the  dirty data set.